fire/SFEquationsMod.F90

```diff
@@ -0,0 +1,296 @@
+    !  Equation A6 in Thonicke et al. 2010
+    !  Rothermel 1972 Eq. 37
+    !
+    
+    ! ARGUMENTS:
+    real(r8), intent(in) :: SAV ! surface area to volume ratio of fuel [/cm]
+    
+    ! CONSTANTS:
+    real(r8), parameter :: a = 0.200395_r8
+    real(r8), parameter :: b = -0.8189_r8
+          
+    if (SAV < nearzero) then
+      OptimumPackingRatio = 0.0_r8 
+    else  
+      OptimumPackingRatio = a*(SAV**b)
+    end if
+          
+    end function OptimumPackingRatio
+    
+    !-------------------------------------------------------------------------------------
+  
+    real(r8) function MaximumReactionVelocity(SAV)
+      !
+      !  DESCRIPTION:
+      !  Calculates maximum reaction velocity in /min
+      !  Reaction velocity that would exist if fuel were free of moisture and 
+      !  contained mineral and alpha cellulose
+      !  From Equation 36 in Rothermel 1972; Fig. 12
+      !
+      
+      ! ARGUMENTS:
+      real(r8), intent(in) :: SAV ! fuel surface area to volume ratio [/cm]
+      
+      MaximumReactionVelocity = 1.0_r8/(0.0591_r8 + 2.926_r8*(SAV**(-1.5_r8)))
+    
+    end function MaximumReactionVelocity
+   
+    !-------------------------------------------------------------------------------------
+  
+    real(r8) function OptimumReactionVelocity(max_reaction_vel, SAV, beta_ratio)
+      !
+      !  DESCRIPTION:
+      !  Calculates optimum reaction velocity in /min
+      !  From Equation 38 in Rothermel 1972; Fig. 11
+      !
+      
+      ! ARGUMENTS:
+      real(r8), intent(in) :: max_reaction_vel ! maximum reaction velocity [/min]
+      real(r8), intent(in) :: SAV              ! fuel surface area to volume ratio [/cm]
+      real(r8), intent(in) :: beta_ratio       ! ratio of packing ratio to optimum packing ratio [0-1]
+      
+      ! LOCALS:
+      real (r8) :: a, a_beta ! intermediate variables
+      
+      ! Equations in Table A1 Thonicke et al. 2010
+      a = 8.9033_r8*(SAV**(-0.7913_r8))
+      a_beta = exp(a*(1.0_r8 - beta_ratio)) 
+      
+      OptimumReactionVelocity = max_reaction_vel*(beta_ratio**a)*a_beta
+      
+    end function OptimumReactionVelocity
+   
+    !-------------------------------------------------------------------------------------
+  
+    real(r8) function MoistureCoefficient(moisture, MEF)
+      !
+      !  DESCRIPTION:
+      !  Calculates the moisture dampening coefficient for reaction intensity
+      !  Based on Equation in table A1 Thonicke et al. 2010. 
+      !
+      
+      ! ARGUMENTS:
+      real(r8), intent(in) :: moisture ! fuel moisture [m3/m3]
+      real(r8), intent(in) :: MEF      ! fuel moisture of extinction [m3/m3]
+      
+      ! LOCALS:
+      real(r8) :: mw_weight ! relative fuel moisture/fuel moisture of extinction
+      
+      ! average values for litter pools (dead leaves, twigs, small and large branches), plus grass
+      mw_weight = moisture/MEF
+      
+      ! moist_damp is unitless
```

Comment:
Should `moist_damp` in comment be `MoistureCoefficient`?

Reply:
yep! thanks